### PR TITLE
Libraries upgrade + test classes fixed

### PR DESCRIPTION
**Pull request by jlannoy**
I'm about to release one or two modules for Ninja, and I noticed some libraries were rather old... I needed to locally update FluentLenium, Jackson, Guava... But I thought it was the right time to make a global update the framework.

So here is a consistant upgrade, with 2 tests classes fixed. Tests are green (in my environment). I tested the snapshot version on some project too, seems ok.

### CODE REVIEW

**Comment by jjlauer** [comment]
This PR looks good to me.  Let's let @raphaelbauer weigh in too.

File: ninja-test-utilities/pom.xml [hunk 1]
```diff
@@ -108,6 +114,12 @@
             <groupId>org.doctester</groupId>
             <artifactId>doctester-core</artifactId>
             <scope>compile</scope>
+            <exclusions>
+            	<exclusion>
+        			<groupId>com.google.guava</groupId>
```
Comment (raphaelbauer) [inline ninja-test-utilities/pom.xml]: tiny.. but here are tabs..

File: ninja-test-utilities/src/main/java/ninja/NinjaFluentLeniumTest.java [hunk 2]
```diff
@@ -20,15 +20,20 @@
...
 public abstract class NinjaFluentLeniumTest extends FluentTest {
 
-	public WebDriver webDriver = new HtmlUnitDriver();
+  /**
+   * @deprecated The driver should be access via {@link #getDriver()}.
+   * Null variable left for deprecation notice only.
+   * @see #getDriver()
+   */
+  @Deprecated
+	public WebDriver webDriver;
```
Comment (raphaelbauer) [inline ninja-test-utilities/src/main/java/ninja/NinjaFluentLeniumTest.java]: Already a mistake in the codebase... but would you mind fixing the tabs to space? thanks!

File: pom.xml [hunk 3]
```diff
@@ -849,6 +849,13 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.google.code.gson</groupId>
+                <artifactId>gson</artifactId>
```
Comment (raphaelbauer) [inline pom.xml]: why do we need gson now?

**Comment by raphaelbauer** [comment]
Really cool! I added tiny remarks... otherwise we can merge asap :) 👍

**Comment by jlannoy** [comment]
I fixed tabs to spaces. 

For Gson, it is used under the hood by doctester, fluentlenium and the htmlunitdriver too (this is why it was scoped as test). There was a problem with the version imported by doctester, and my exclusion tag was not working. But after a second check :  it was a really stupid mistake (I inverted groupId and artifactId). ... I was able to remove the Gson mention from the master pom ;)

**Comment by raphaelbauer** [comment]
Awesome! Good work!

**Pull request merged**